kafka-ui-react-app/src/lib/__test__/yupExtended.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { isValidEnum, isValidJsonObject } from 'lib/yupExtended';
+import { isValidEnum, isValidJsonObject, topicFormValidationSchema } from 'lib/yupExtended';
```

Comment:
please check the linter build and test warnings

Reply:
Has been fixed.